app/views/layouts/application.html.erb

```diff
@@ -12,4 +12,18 @@
   <body>
     <%= yield %>
   </body>
+
+  <!-- <footer>
+    <h3><%= @repo.repo_name %></h3>
+
+    <h3>Contributors</h3>
+    <% @contributors.each do |contributor| %>
+      <% if contributor.user_name == "Sierra-T-9598" || contributor.user_name == "jbreit88" || contributor.user_name == "karanm645" || contributor.user_name == "jgrazulis" %>
```

Comment:
Is there a way to do this without hard-coding our names? I know it may be bc there are other contributors we don't want listed...

Reply:
Yeah, it is because there are other contributors that we don't want listed. We could possibly filter the names by people who have commits in the repo history. I am also fine not hardcoding anything and having other contributors names show up on the page.